br/pkg/lightning/importer/table_import_test.go

```diff
@@ -99,6 +99,10 @@ func (s *tableRestoreSuiteBase) setupSuite(t *testing.T) {
 	core, err := ddl.MockTableInfo(se, node.(*ast.CreateTableStmt), 0xabcdef)
 	require.NoError(t, err)
 	core.State = model.StatePublic
+	core.TiFlashReplica = &model.TiFlashReplicaInfo{
```

Comment:
Please add a test case for multiple tables with different tiflash replicas.

Reply:
added in https://github.com/pingcap/tidb/pull/41943/commits/931f006043591ae0ec7cb22e190013570b15cfad